### ISSUE TITLE
Cannot expand snippets with pre_expand

### ISSUE THREAD

**Issue by szsdk** [neoclide/coc-snippets#205]
For example, the tex snippets for generating table: `gentbl3x5` does not. After  '<Plug>(coc-snippets-expand)', the input just disappears. But other snippets without 'pre_expadn' or 'post_jump' work fine. Is it a feature? I have no idea how to debug this problem.

**Comment by chemzqm** [comment]
We don't have any support for ultisnip hooks, as mentioned in readme.

**Comment by SeniorMars** [comment]
Can I ask what the problem with ultisnip hooks is? Is there a lot of complexity involved with the feature?

**Comment by chemzqm** [comment]
> Can I ask what the problem with ultisnip hooks is? Is there a lot of complexity involved with the feature?

Yes, lots of problems with those hooks, should be fixed by https://github.com/neoclide/coc.nvim/pull/5282
Use latest coc.nvim and coc-snippets to use  UltiSnips actions.
